indico/htdocs/sass/base/_layout.scss

```diff
@@ -136,7 +136,14 @@ body > * {
     }
 }
 
-/* Flexbox layout */
+
+// -- Flexbox layout -----------------------------------------------------------
```

Comment:
Maybe the presence of such a comment is a good indicator that the block below belongs into a separate file?